Webpage/Code/Comments/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FC, useEffect, useState, useRef } from "react"
+
+import { Container as ContainerStyle } from "../Card/Styles.css"
+import * as Styles from "./Styles.css"
+
+interface Comment {
+  username: string
+  message: string
+  time: number
+}
+
+const headers = {
+  'Accept': 'application/json',
+  'Content-Type': 'application/json;charset=UTF-8'
+}
+
+const Comments: FC = () => {
+  const [comments, updateComments] = useState([] as Array<Comment>)
+
+  async function getComments() {
+    const response = await fetch("/get_comments", { method: "POST" })
+    const comments = (await response.json()) as Array<Comment>
+    updateComments(comments)
+  }
+
+  useEffect(() => {
+    getComments()
+    const id = setInterval(getComments, 3000)
+    return () => clearInterval(id)
+  }, [])
+
+  const form = useRef<HTMLFormElement>(null);
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault()
+    const elements = form.current?.elements
+    if (!elements) return
+
+    const username = elements["username"].value as string
+    const message = elements["message"].value as string
+    elements["username"].value = ""
+    elements["message"].value = ""
+
+    const body = JSON.stringify({username,  message})
+
+    const response = await fetch('/create_comment', { method: 'POST', headers, body})
+    const serverSaid = (await response.json())
+    console.log(serverSaid)
```

Comment:
Do you want to keep this logging in the code after submitting?

Reply:
Hahaha, no, totally forgot to delete that at the end 😝